hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/RowCounter.java

```diff
@@ -388,4 +460,9 @@ protected CommandLineParser newParser() {
     return new RowCounterCommandLineParser();
   }
 
+  // Visible for testing
+  public Job getMapReduceJob() {
```

Comment:
if we are expose this please refer how we are exposing method visible for testing at other places in codebase. also i think package visibility would be good enough here ?